pkg/operator/configobservation/etcd/observe_etcd.go

```diff
@@ -63,6 +63,13 @@ func ObserveStorageURLs(genericListers configobserver.Listers, recorder events.R
 				errs = append(errs, addressErr)
 				continue
 			}
+			// the installer uses dummy addresses in the subnet `192.0.2.` for host-etcd endpoints
+			// this check see if etcd-bootstrap is populated with a real ip address and uses it
+			// instead of FQDN
+			if address.Hostname == "etcd-bootstrap" && !strings.Contains(address.IP, "192.0.2") {
```

Comment:
could we be more explicit using `strings.HasPrefix`

https://play.golang.org/p/zhN4gUkjV_3

Reply:
We will need to rethink this if the endpoint address is IPv6 should we add TODO?

Reply:
I guess since the ep is static in the check populated by installer probably not a big deal.

Reply:
done